src/pytest_benchmark/utils.py

```diff
@@ -186,20 +186,26 @@ def get_commit_info(project_name=None):
                 dirty = True
                 desc.pop()
             commit = desc[-1].strip('g')
+            commit_time = check_output('git show -s --pretty=format:"%cI"'.split(),
+                                       universal_newlines=True).strip().strip('"')
         elif in_any_parent('.hg'):
             desc = check_output('hg id --id --debug'.split(), universal_newlines=True).strip()
             if desc[-1] == '+':
                 dirty = True
             commit = desc.strip('+')
+            commit_time = check_output('hg tip --template "{date|rfc3339date}"'.split(),
+                                       universal_newlines=True).strip().strip('"')
         return {
             'id': commit,
+            'time': commit_time,
             'dirty': dirty,
             'project': project_name,
             'branch': branch,
         }
     except Exception as exc:
         return {
             'id': 'unknown',
+            'time': '1970-01-01T01:00:00+01:00',
```

Comment:
I would put a `None` instead as an actual date would create a confusing situation where whoever reads the json can't really be sure if the commit was in the past, the commit has wrong date or date simply wasn't available.